### PR TITLE
update polkadot-v0.9.26

### PR DESCRIPTION
**Pull request by zqhxuyuan**

### CODE REVIEW

File: traits/src/location.rs [hunk 1]
```diff
@@ -82,7 +82,13 @@ pub trait RelativeLocations {
 
 impl RelativeLocations for MultiLocation {
 	fn sibling_parachain_general_key(para_id: u32, general_key: Vec<u8>) -> MultiLocation {
-		MultiLocation::new(1, X2(Parachain(para_id), GeneralKey(general_key)))
+		MultiLocation::new(
+			1,
+			X2(
+				Parachain(para_id),
+				GeneralKey(general_key.to_vec().try_into().expect("less than length limit; qed")),
```
Comment (xlc) [inline traits/src/location.rs:89]: I don't think this is safe. We should change the `general_key` type

**Review by xlc — APPROVED** [review]

**Comment by Kailai-Wang** [comment]
Sorry, but is the deletion of `polkadot-v0.9.26` branch intended? Shall we depend on the master branch in case we want to upgrade the deps to v0.9.26?

**Comment by xlc** [comment]
sorry @Kailai-Wang restored.

**Comment by Kailai-Wang** [comment]
All good, thanks @xlc !

**Pull request merged**